### DISCUSSION TITLE
Did I miss the announced change to Apache 2.0 for Tina Data Layer?

### DISCUSSION THREAD

**Discussion by ryangorley** [tinacms/tinacms#4199] [Help] [answered]
Just a few weeks ago, I decided not to use Tina because of the dependency on the data layer that wasn't open source but was "source available" with user restrictions. I'm not mistaken because there are still references to these license restrictions (see: https://github.com/tinacms/tinacms/discussions/3589). Note that the license pages linked to from that discussion are missing, and the blog article (see: https://tina.io/blog/self-hosted-datalayer/) released around the same time has since been updated to say the data layer is Apache 2.0 (though Google has it cached saying otherwise). Now, this is welcome news and is cause for me to give Tina another look. But did I miss an announcement about this change? Can I trust my eyes, or am I missing something here?

**Comment by scottgallant** [comment, marked as answer]
Hey @ryangorley, you can believe your eyes 😀 

We didn't make the official announcement yet because we have more changes coming that will make self hosting better. We just wanted to get the license changed asap for people (like yourself) who are stumbling on Tina and want a fully open-source solution. 

We should be ready to announce in a week or so...until then please keep it secret 🤫 😉

**Reply by ryangorley** [reply]
Awesome. Excited to learn more. Thanks!
